cluster/images/kubelet/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM centos
```

Comment:
Is there a way to verify whether util-linux via this list would contain nsenter?